sfge/src/test/java/com/salesforce/graph/source/supplier/PageReferenceSupplierTest.java

```diff
@@ -0,0 +1,50 @@
+package com.salesforce.graph.source.supplier;
+
+import com.salesforce.collections.CollectionUtil;
+import org.junit.jupiter.api.Test;
+
+public class PageReferenceSupplierTest extends BaseSourceSupplierTest {
+
+    /** Non-test methods that return page references should be returned by this supplier. */
+    @Test
+    public void supplierLoadsPageReferenceMethods() {
```

Comment:
Equivalent to `PathEntryPointUtilTest#getPathEntryPoints_includesPageReferenceMethods` and `PathEntryPointUtilTest#testGetPageReferenceMethods`.